tests/test_dsrmanager.py

```diff
@@ -0,0 +1,102 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2020 kentonprescott
```

Comment:
The copyright should be changed to Copyright (C) 2020 Maker Ecosystem Growth Holdings, INC.